webpages/profile/metaanalysis.html

```diff
@@ -613,29 +614,51 @@ <h1 class="title editing oneline" contenteditable placeholder="enter a short nam
       .positioningbutton line {
         stroke-width: 1.5px;
       }
+
+      .forest-group .group-heading text.label {
+        font-weight: bold;
+        font-size: 120%;
+      }
+
+      .summary .sumname.sumtotal {
+        font-weight: bold;
+        font-size: 125%;
+      }
     </style>
 
     <g transform="translate(10,0)" class="headings">
-      <text class="or">OR</text>
-      <text class="lcl">LCL,</text>
-      <text class="ucl">UCL</text>
+      <text class="or lcl ucl">OR [LCL, UCL]</text>
       <text class="wt">WT</text>
+      <text class="wtg">, WTG</text>
     </g>
 
     <template class="experiment">
       <g class="experiment"><!-- should get transform="translate(padding,currY++)" -->
         <text class="expname">error</text>
-        <text class="or">invalid</text>
-        <text class="lcl">n/a,</text>
-        <text class="ucl">n/a</text>
+        <text class="or lcl ucl">invalid [n/a, n/a]</text>
         <text class="wt">n/a</text>
+        <text class="wtg">, n/a</text>
```

Comment:
does this mean we are adding a ", " in the code above? It would then be better to have three <text> elements, only two of which we are changing.